src/Uno.UWP/Media/Playback/MediaPlayer.iOS.cs

```diff
@@ -397,7 +397,10 @@ private void OnSeekCompleted(bool finished)
 
 		internal void UpdateVideoGravity(AVLayerVideoGravity gravity)
 		{
-			_videoLayer.VideoGravity = gravity;
+			if (_videoLayer != null)
```

Comment:
could also be shorten to:

    _videoLayer?.VideoGravity = gravity;

Reply:
There's a specific style check error in error that stopped me from doing that